libddwaf-1.6.1-darwin-arm64.tar.gz.sha256

```diff
@@ -0,0 +1 @@
+a1518e7a4d93842e1edb07b7b878b36063b3cd073b02e02610d3e5274378cb54  libddwaf-1.6.1-darwin-arm64.tar.gz
```

Comment:
You'll want to remove this before merge, I'm guessing it got included accidentally.

Reply:
oops, yes, sorry!